auth0/src/main/java/com/auth0/android/authentication/storage/CredentialsManager.java

```diff
@@ -100,34 +93,65 @@ public void saveCredentials(@NonNull Credentials credentials) {
      *
      * @param callback the callback that will receive a valid {@link Credentials} or the {@link CredentialsManagerException}.
      */
-    public void getCredentials(@NonNull final BaseCallback<Credentials, CredentialsManagerException> callback) {
+    public void getCredentials(@NonNull BaseCallback<Credentials, CredentialsManagerException> callback) {
```

Comment:
Mark the `callback` parameter as `final`, like it is in the overload.

Reply:
the `final` modifier is only required when passed to a nested body / lambda. In this case, that's no longer required.